docs/setup.md

```diff
@@ -34,13 +34,13 @@ Get your OpenAI API key from: [https://platform.openai.com/account/api-keys](htt
 ### Set up with Docker
 
 1. Make sure you have Docker installed, see [requirements](#requirements)
-2. Pull the latest image from [Docker Hub]
+2. Create a project directory for Auto-GPT
 
         :::shell
```

Comment:
":::shell", should this be visible here? codeblock marker can be mistaken for code